src/common/components/elements/ActivityContent.tsx

```diff
@@ -133,7 +133,7 @@ export const ActivityContent = ({ publicKey }: { publicKey: PublicKey | null })
     return (
       <ItemText>
         <b>{getDisplayName(twitterHandle, publicKey)}</b> bid{' '}
-        {(bid.lastBidAmount ?? 0) / LAMPORTS_PER_SOL} SOL on <b>{bid.listing?.nfts?.[0]?.name}</b>
+        {(bid.lastBidAmount ?? 0) / LAMPORTS_PER_SOL} on <b>{bid.listing?.nfts?.[0]?.name}</b>
```

Comment:
This upgrade is blocked by https://github.com/holaplex/solana-indexer/pull/171, once merged. We can show the mint of the token that the bid was in.

Reply:
and/or wum.bo ;)

Reply:
in wumbo's case it's onchain afaik as meta